globalvalues.py

```diff
@@ -24,6 +24,8 @@
 DEFAULT_PUBLICKEY = '/home/pi/config/id_rsa_lbl.pub'
 DEFAULT_LOGFILE = '/home/pi/debug.log'
 DEFAULT_HOSTNAME = 'dosenet.dhcp.lbl.gov'
+DEFAULT_DATALOG = '/home/pi/data-log.txt'
+DEFAULT_TEST_DATALOG = '/home/pi/data-log-testfile.txt'
```

Comment:
`DEFAULT_TEST_DATALOG` is nice for running on the Pi, but the test code should also run on other machines (while skipping certain tests). So it would be better to put the test data log in the current working directory rather than an absolute path. It could just be defined in `tests.py` rather than in `globalvalues.py`.